jwst/regtest/test_miri_lrs_slitless_tso1.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from astropy.io.fits.diff import FITSDiff
+
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+
+DATASET_ID = "det_image_MIRIMAGE_P750Lexp1"
```

Comment:
I think it would be best to use standard DMS style file names for the input and output products, because the infrastructure within ``stpipe`` is setup to look for and do certain things with the various parts of the standard file names. Even if you just have to make up something, like jw1234500101_etc. Better yet, since it appears that these data came from MIRISim, the data array itself should be copied into an actual MIRI LRS slitless _uncal file that was produced by DMS, so that we know for sure that everything about the file structure, organization, and header keywords is completely as expected from DMS.

Reply:
@bushhouse not sure what you mean by "the data array itself should be copied into an actual MIRI LRS slitless_uncal file." I had to take a simulated data set read it and and chop out 90% of the file.  Where can I get ahold of a DMS produced MIRI LRS slitless_cal file ?

Reply:
Since this test runs the ``calwebb_tso1`` pipeline I assume that the simulated data is in raw 4-D format, correct? If so, then you can just copy the 4-D data array from "det_image_MIRIMAGE_P750Lexp1" into any of the MIRI LRS-SLITLESS uncal files produced by DMS B7.4 that are sitting in "/grp/jwst/ins/mary/b7.4_cal14_full/", such as:

/jw00859/jw00859001001_03102_00001_mirimage_uncal.fits
/jw00623/jw00623024001_02103_00001_mirimage_uncal.fits
/jw00623/jw00623026001_03103_00010_mirimage_uncal.fits
etc.

You may have to modify a few header keyword values for things like NGROUPS and NINTS to make them match the simulated data.

Reply:
@hbushouse well I do not have permission to get data in that directory
Who should I contact so I can get permission ?

Reply:
@mromelfanger In the meantime let me know what file(s) you'd like and I can grab them for you.

Reply:
@hbushouse I just need any one of the files you suggested.

Reply:
You can grab:
/grp/jwst/ssb/bushouse/jwst_data/MIRI/LRS/SLITLESS/jw00623026001_03106_00005_mirimage_uncal.fits